README.md

```diff
@@ -124,7 +124,7 @@ You will be building your "OOP school library" app for more than a week. Here is
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Cool, Thank you for the advice